pkg/virtctl/create/vm/vm.go

```diff
@@ -243,6 +258,9 @@ func (c *createVM) usage() string {
   # Create a manifest for a VirtualMachine with a cloned DataSource and inferred instancetype and preference
   {{ProgramName}} create vm --volume-datasource=src:my-annotated-ds --infer-instancetype --infer-preference
 
+  # Create a manifest for a VirtualMachine with multiple volumes and specified boot order
+  {{ProgramName}} create vm --volume-containerdisk=src:my.registry/my-image:my-tag --volume-datasource=src:my-ds,bootorder:1
```

Comment:
Can't we simply use the volume name instead on the need to specify the type of volume? The name should be unique

Reply:
The flags are not only setting the boot order of a volume but also add the volume itself. So I don't think this can be simplified any more.